libsel4camkes/docs/pid.md

```diff
@@ -0,0 +1,14 @@
+<!--
+     Copyright 2021, Data61, CSIRO (ABN 41 687 119 230)
+
+     SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# PID
+
+This is a constant variable that tracks the ID of the CAmkES component.
+Although CAmkES or seL4 does not have a concept of processes, these variable is
+used to maintain compatibility with parts of the POSIX standard. Note that the
```

Comment:
Do this mean parts of POSIX are supported?

Reply:
yes